api/metrics_service.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "google/api/annotations.proto";
+import "metrics.proto";
+
+// Service for streaming metrics to connected end point.
+service MetricsService {
+  rpc StreamMetrics(stream StreamMetricsMessage) returns (StreamMetricsResponse) {
+  
+  }
+}
+
+message StreamMetricsResponse {}
+
+message StreamMetricsMessage {
+  repeated io.prometheus.client.MetricFamily proxy_metrics = 1;
```

Comment:
envoy_metrics. We don’t care if this is proxy or sidecar or LB or anything :).

Reply:
will change